### ISSUE TITLE
Loki server error: msg="error syncing tables" err="InvalidParameter: 2 validation error(s) found.\n- minimum field value of 1, UpdateTableInput.ProvisionedThroughput.ReadCapacityUnits.\n- minimum field value of 1, UpdateTableInput.ProvisionedThroughput.WriteCapacityUnits.\n"

### ISSUE THREAD

**Issue by dudusakharovich** [grafana/loki#806]
**Describe the bug**
Tried to configure AWS S3 & DynamoDB as storage config.
Received the following error while running loki server:
msg="error syncing tables" err="InvalidParameter: 2 validation error(s) found.\n- minimum field value of 1, UpdateTableInput.ProvisionedThroughput.ReadCapacityUnits.\n- minimum field value of 1, UpdateTableInput.ProvisionedThroughput.WriteCapacityUnits.\n"

**Expected behavior**

**Environment:**
Installed on AWS EC2-Instance from source.

**Comment by sandlis** [comment]
@dudusakharovich Can you please share your config for table manager or check whether you are setting read/write throughput to 0 by any chance?

**Comment by yeya24** [comment]
I met the same problem. Seems the ksonnet configs for AWS is not right. My configuration is 

```
    table_manager:
        chunk_tables_provisioning:
            inactive_read_throughput: 0
            inactive_write_throughput: 0
            provisioned_read_throughput: 0
            provisioned_write_throughput: 0
        index_tables_provisioning:
            inactive_read_throughput: 0
            inactive_write_throughput: 0
            provisioned_read_throughput: 0
            provisioned_write_throughput: 0
        retention_deletes_enabled: false
        retention_period: 0
```

table-manager error log is 

```
InvalidParameter: 2 validation error(s) found.\n- minimum field value of 1, CreateTableInput.ProvisionedThroughput.ReadCapacityUnits.\n- minimum field value of 1, CreateTableInput.ProvisionedThroughput.WriteCapacityUnits.\n; InvalidParameter: 2 validation error(s) found.\n- minimum field value of 1, CreateTableInput.ProvisionedThroughput.ReadCapacityUnits.\n- minimum field value of 1, CreateTableInput.ProvisionedThroughput.WriteCapacityUnits.\n; InvalidParameter: 2 validation error(s) found.\n- minimum field value of 1, CreateTableInput.ProvisionedThroughput.ReadCapacityUnits.\n- minimum field value of 1, CreateTableInput.ProvisionedThroughput.WriteCapacityUnits.\n; InvalidParameter: 2 validation error(s) found.\n- minimum field value of 1, CreateTableInput.ProvisionedThroughput.ReadCapacityUnits.\n- minimum field value of 1, CreateTableInput.ProvisionedThroughput.WriteCapacityUnits.\n; InvalidParameter: 2 validation error(s) found.\n- minimum field value of 1, CreateTableInput.ProvisionedThroughput.ReadCapacityUnits.\n- minimum field value of 1, CreateTableInput.ProvisionedThroughput.WriteCapacityUnits.\n; InvalidParameter: 2 validation error(s) found.\n- minimum field value of 1, CreateTableInput.ProvisionedThroughput.ReadCapacityUnits.\n- minimum field value of 1, CreateTableInput.ProvisionedThroughput.WriteCapacityUnits.\n; InvalidParameter: 2 validation error(s) found.\n- minimum field value of 1, CreateTableInput.ProvisionedThroughput.ReadCapacityUnits.\n- minimum field value of 1, CreateTableInput.ProvisionedThroughput.WriteCapacityUnits.\n; InvalidParameter: 2 validation error(s) found.\n- minimum field value of 1, CreateTableInput.ProvisionedThroughput.ReadCapacityUnits.\n- minimum field value of 1, CreateTableInput.ProvisionedThroughput.WriteCapacityUnits.\n; InvalidParameter: 2 validation error(s) found.\n- minimum field value of 1, CreateTableInput.ProvisionedThroughput.ReadCapacityUnits.\n- minimum field value of 1, CreateTableInput.ProvisionedThroughput.WriteCapacityUnits.\n; InvalidParameter: 2 validation error(s) found.\n- minimum field value of 1, CreateTableInput.ProvisionedThroughput.ReadCapacityUnits.\n- minimum field value of 1, CreateTableInput.ProvisionedThroughput.WriteCapacityUnits.\n; InvalidParameter: 2 validation error(s) found.\n- minimum field value of 1, CreateTableInput.ProvisionedThroughput.ReadCapacityUnits.\n- minimum field value of 1, CreateTableInput.ProvisionedThroughput.WriteCapacityUnits.\n; In
```

**Comment by ruanbekker** [comment]
@yeya24 I had the same issue, I deleted my dynamodb table, and started loki so that it can provision the table for me.

Ensure you have these permissions in place:
- https://github.com/grafana/loki/tree/master/docs/operations/storage#dynamodb

My config which i'm testing with:

```
table_manager:
  chunk_tables_provisioning:
    inactive_read_throughput: 1
    inactive_write_throughput: 1
    provisioned_read_throughput: 5
    provisioned_write_throughput: 5
  index_tables_provisioning:
    inactive_read_throughput: 1
    inactive_write_throughput: 1
    provisioned_read_throughput: 5
    provisioned_write_throughput: 5
  retention_deletes_enabled: false
  retention_period: 0
``` 

Logs on loki:

```
$ journalctl -u loki -f
-- Logs begin at Thu 2020-04-23 11:24:28 UTC. --
...
Apr 23 13:59:37 ip-172-31-37-225 loki[2535]: level=info ts=2020-04-23T13:59:37.734389562Z caller=main.go:74 msg="Starting Loki" version="(version=1.4.1, branch=HEAD, revision=2445ccd2)"
Apr 23 13:59:37 ip-172-31-37-225 loki[2535]: level=info ts=2020-04-23T13:59:37.822284167Z caller=table_manager.go:394 msg="creating table" table=loki-logging-index
Apr 23 14:01:37 ip-172-31-37-225 loki[2535]: level=info ts=2020-04-23T14:01:37.728552705Z caller=table_manager.go:247 msg="synching tables" expected_tables=1
Apr 23 14:01:38 ip-172-31-37-225 loki[2535]: level=info ts=2020-04-23T14:01:38.736211246Z caller=table_manager.go:449 msg="provisioned throughput on table, skipping" table=loki-logging-index read=5 write=5
```

**Comment by yeya24** [comment]
@ruanbekker Hello, thanks for your reply. It is really helpful!
But I am curious that how did you choose the `provisioned_read_throughput` and `provisioned_write_throughput` parameter? Is there any recommendation or metrics for dynamodb to tune my configuration?

**Comment by ruanbekker** [comment]
Hey @yeya24 , I'm still testing it out, so I went with a number and then decided to monitor the metrics afterwards. 

As DynamoDB's Read IOP is up to 4KB and Write IOP 1KB, I made a guess that 40KB/s and 5KB/s would be okay for now, as I'm not pushing and reading log data too heavily at the moment. 

I think when the data increases over time, I will be able to get a feel what the metric should be. This is for the last 2 days:

<img width="795" alt="image" src="https://user-images.githubusercontent.com/567298/80275629-e7541680-86e2-11ea-8acf-c0ab8c4e07b3.png">
